.github/workflows/test-phpunit.yml

```diff
@@ -110,21 +110,12 @@ jobs:
           DB: ${{ matrix.db-platforms }}
           TERM: xterm-256color
 
-      - name: Run Coveralls
+      - if: matrix.php-versions == '7.4'
+        name: Run Coveralls
         run: |
           composer global require php-coveralls/php-coveralls:^2.4
           php-coveralls --coverage_clover=build/logs/clover.xml -v
         env:
           COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           COVERALLS_PARALLEL: true
           COVERALLS_FLAG_NAME: PHP ${{ matrix.php-versions }}
-
-  coveralls-finish:
-    needs: [tests]
-    runs-on: ubuntu-latest
-    steps:
-      - name: Coveralls Finished
-        uses: coverallsapp/github-action@master
-        with:
-          github-token: ${{ secrets.github_token }}
-          parallel-finished: true
```

Comment:
We have to restore it. Since our build run in parallel Coveralls needs to know when the job is finished. Now the page with results says: "Pending completion".

Reply:
implemented with add `- if: matrix.php-versions == '7.4'` check

Reply:
I don't think there is a `matrix` variable available here. Also, it's not needed since it runs only once for all tests.

Reply:
ok, removed if check.

Reply:
Thanks.